src/assets/css/splash.scss

```diff
@@ -24,6 +24,11 @@
 
 .splash--item {
     min-width: 50vw;
+    min-height: 100vh;
+    display: inline-flex;
+    flex-direction: row;
+    align-items: center;
+
```

Comment:
Something you did over here misaligned the text :see_no_evil: 

![DeepinScreenshot_20190801093820](https://user-images.githubusercontent.com/7110169/62280489-ecca4c00-b443-11e9-91b0-8b28fcb86ec7.png)

Also, something is _just_ over 100vw, which means that nasty horizontal scrolling bar is showing.